### PR TITLE
Update kang.py

### PR DESCRIPTION
**Pull request by nealgosalia**
Simplify code

### CODE REVIEW

**Comment by rking32** [comment]
@nealgosalia thanks for your PR
this worked for you, right ?

File: userge/plugins/fun/kang.py [hunk 1]
```diff
@@ -209,21 +208,10 @@ def resize_photo(photo: str) -> str:
...
-            size1new = size1 * scale
-            size2new = 512
-        size1new = math.floor(size1new)
-        size2new = math.floor(size2new)
-        sizenew = (size1new, size2new)
-        image = image.resize(sizenew)
+    if max(image.width, image.height) < 512:
+        scale = 512 / max(image.width, image.height)
+        new_size = (int(image.width * scale), int(image.height * scale))
+        image = image.resize(new_size, Image.ANTIALIAS)
```
Comment (rking32) [inline userge/plugins/fun/kang.py:214]: what is that `Image.ANTIALIAS` ?

Comment (nealgosalia) [reply userge/plugins/fun/kang.py:214]: @rking32 It's an image resampling method. It's better for upscaling compared to the default one.

Comment (rking32) [reply userge/plugins/fun/kang.py:214]: great

**Pull request merged**